rustls/src/key_log_file.rs

```diff
@@ -106,6 +116,16 @@ impl KeyLog for KeyLogFile {
     }
 }
 
+impl Debug for KeyLogFile {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        if let Ok(key_log_file) = self.0.try_lock() {
+            write!(f, "{:?}", key_log_file)
+        } else {
+            write!(f, "KeyLogFile {{ <locked> }}")
+        }
```

Comment:
Nit: I'd write it as a match, since it's one less line.

Reply:
Fixed.